gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/java/io/gravitee/repository/jdbc/management/JdbcPortalPageRepository.java

```diff
@@ -77,4 +78,38 @@ public List<PortalPage> findByIds(List<String> ids) {
             return List.of();
         }
     }
+
+    @Override
+    public List<PortalPage> findByIdsWithExpand(List<String> ids, List<String> expand) {
+        if (ids == null || ids.isEmpty()) {
+            return List.of();
+        }
+        try {
+            var selected = new LinkedHashSet<String>();
+            selected.add("id");
+            if (expand != null) {
+                for (String e : expand) {
+                    if (e != null) {
+                        var col = e.trim();
+                        if (!col.isEmpty()) {
+                            selected.add(col);
+                        }
+                    }
+                }
+            }
```

Comment:
would it work with something simplier like this ?
```suggestion
    var selected = new LinkedHashSet<String>();
    selected.add("id");
    if (expand != null) {
        // This is the clean, efficient way
        selected.addAll(expand);
    }    
```

Reply:
changed to something similar since now we are not using List<String> instead using List<enumforExpands>